xml/System/Memory`1.xml

```diff
@@ -311,9 +380,9 @@
         <Parameter Name="array" Type="T[]" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="array">The array to convert.</param>
+        <summary>Defines an implicit conversion of an array to a <see cref="T:System.Memory`1" /> object.</summary>
+        <returns>The converted object.</returns>
```

Comment:
`To be added` remarks here and elsewhere

Reply:
That's fine @ahsonkhan. It's the default value and is not rendered on docs.